sdk/digitaltwins/Azure.DigitalTwins.Core/README.md

```diff
@@ -6,26 +6,38 @@ This library provides access to the Azure Digital Twins service for managing twi
 
 ## Getting started
```

Comment:
I recommend getting a markdown linter for VS Code and using it.

One rule us to always leave blank lines around headers, lists, and code blocks.

Reply:
I will get it right away and fix the formatting